docs/utils/nft-borsh.ts

```diff
@@ -93,9 +105,10 @@ export const NFTOKEN_MINTLIST_ACCOUNT = new FixableGlowBorsh<{
     ["treasury_sol", GlowBorsh.address],
     ["go_live_date", GlowBorsh.timestamp],
     ["price", GlowBorsh.solAmount],
-    ["minting_order", u8], // TODO: Figure out what this should be to be an enum
+    // @ts-ignore
+    ["minting_order", fixedScalarEnum(MintingOrder)],
     ["collection", GlowBorsh.address],
-    ["metadata_url", FixableGlowBorsh.utf8String],
+    ["metadata_url", GlowBorsh.utf8String(96)],
```

Comment:
For more context, we use `GlowBorsh.utf8String(96)` here because the `metadata_url` on a Mintlist should be a fixed width so we can easily calculate where to find the `mint_infos`.